src/tensorlake/applications/remote/manifests/function.py

```diff
@@ -103,10 +110,14 @@ def _type_hint_json_schema(type_hint) -> Dict[str, str]:
     if type_hint == Any:
         return {"type": "string", "description": "Any type"}
 
-    # Handle typing generics like List, Dict, etc. first
+    # Handle Pydantic BaseModel first
+    if inspect.isclass(type_hint) and issubclass(type_hint, BaseModel):
+        if hasattr(type_hint, "model_json_schema"):
+            return type_hint.model_json_schema()
```

Comment:
That a nice method. I'm wondering if in case of non-model type hint we can create a Model class dynamically and just call its model_json_schema()["field_name"]. No need to implement this right now, just as an idea.